### ISSUE TITLE
Typo in alsabar.lua?

### ISSUE THREAD

**Issue by wimstefan** [copycat-killer/lain#252]
There might be a small typo in alsabar.lua at line 166 ...
Isn't that supposed to be `alsabar.update()` ?

I made a diff for that just in case ...

``` diff
diff --git a/widgets/alsabar.lua b/widgets/alsabar.lua
index 5a54829..cdbefd2 100644
--- a/widgets/alsabar.lua
+++ b/widgets/alsabar.lua
@@ -163,7 +163,7 @@ local function worker(args)
           end),
           awful.button({}, 2, function()
                                                awful.util.spawn(string.format("%s set %s 100%%", alsabar.cmd, alsabar.channel))
-            pulsebar.update()
+            alsabar.update()
           end),
           awful.button({}, 3, function()
             awful.util.spawn(string.format("%s set %s toggle", alsabar.cmd, alsabar.togglechannel or alsabar.channel))
```

**Comment by aajjbb** [comment]
Yes, it is. I've addressed it in #253

**Comment by wimstefan** [comment]
Thank you! I was in doubt if it's worth to create a PR ...
Next time I'll know ;-)
